_inc/client/at-a-glance/index.jsx

```diff
@@ -1,16 +1,17 @@
 /**
  * External dependencies
  */
-import React from 'react';
+import React, { Component } from 'react';
 import { connect } from 'react-redux';
 import { translate as __ } from 'i18n-calypso';
 import analytics from 'lib/analytics';
 
 /**
  * Internal dependencies
  */
+import { ModuleSettingsForm as moduleSettingsForm } from 'components/module-settings/module-settings-form';
 import DashSectionHeader from 'components/dash-section-header';
-import DashStats from './stats';
+import DashStats from './stats/index.jsx';
```

Comment:
We technically don't but my text editor was complaining about it not being there /shrug